### PR TITLE
Add info about use_mkldnn to Dygraph Resnet, Mobilenet logs

### PR DESCRIPTION
**Pull request by arlesniak**
Added more verbose info to Dygraph Resnet and Mobilenet logs about use_mkldnn
Additionally added verbose dump of params to Dygraph Resnet log.

### CODE REVIEW

**Review by jczaja — APPROVED** [review]
LGTM

**Review by jczaja — APPROVED** [review]

File: dygraph/mobilenet/utils/utility.py [hunk 1]
```diff
@@ -53,6 +53,21 @@ def print_arguments(args):
...
+    print("\n-------------  Environment variables  --------------")
+    print("%25s : %s" %
+          ('FLAGS_use_mkldnn', os.environ.get('FLAGS_use_mkldnn', False)))
+    mkldnn_ops_on = os.environ.get('FLAGS_tracer_mkldnn_ops_on', None)
+    mkldnn_ops_off = os.environ.get('FLAGS_tracer_mkldnn_ops_off', None)
+    if mkldnn_ops_on or mkldnn_ops_off:
+        """ 
+        FLAGS_tracer_mkldnn_ops_on is list of ops to be activated default empty means all mkldnn ops 
+        FLAGS_tracer_mkldnn_ops_off default empty means no exceptions from the on list
+        On list has priority over off list
```
Comment (wojtuss) [inline dygraph/mobilenet/utils/utility.py]: Please add punctuation, as the message is not clear.

File: dygraph/resnet/train.py [hunk 2]
```diff
@@ -34,6 +36,42 @@
...
+    print("\n-------------  Environment variables  --------------")
+    print("%25s : %s" %
+          ('FLAGS_use_mkldnn', os.environ.get('FLAGS_use_mkldnn', False)))
+    mkldnn_ops_on = os.environ.get('FLAGS_tracer_mkldnn_ops_on', None)
+    mkldnn_ops_off = os.environ.get('FLAGS_tracer_mkldnn_ops_off', None)
+    if mkldnn_ops_on or mkldnn_ops_off:
+        """ 
+        FLAGS_tracer_mkldnn_ops_on is list of ops to be activated default empty means all mkldnn ops 
+        FLAGS_tracer_mkldnn_ops_off default empty means no exceptions from the on list
+        On list has priority over off list
```
Comment (wojtuss) [inline dygraph/resnet/train.py]: Ditto.

Comment (arlesniak) [reply dygraph/resnet/train.py]: Done

**Comment by arlesniak** [comment]
@luotao1 Could you please start your review?

**Review by wojtuss — APPROVED** [review]
LGTM

**Comment by arlesniak** [comment]
@luotao1 , could you please add your review?

**Review by luotao1 — COMMENTED** [review]
Is this too complex since we should update each `train.py` in all repos?
How about update the logs in https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/framework/executor.cc#L549

```
void Executor::EnableMKLDNN(const ProgramDesc& program) {
#ifdef PADDLE_WITH_MKLDNN
  VLOG(3) << "use_mkldnn=True";
  for (size_t bid = 0; bid < program.Size(); ++bid) {
    auto* block = const_cast<ProgramDesc&>(program).MutableBlock(bid);
    for (auto* op : block->AllOps()) {
      if (op->HasAttr("use_mkldnn")) {
        op->SetAttr("use_mkldnn", true);
      }
    }
  }
```
Change the `VLOG(3) << "use_mkldnn=True";` to `LOG(INFO) << "use_mkldnn=True"` and add some comments?

**Comment by lidanqing-vv** [comment]
Later, luotao said we could change "use_mkldnn=True" to more descriptive sentence so that it could be obvious in the log

**Comment by arlesniak** [comment]
@luotao1 Thank you very much for your comment and suggestion. I understand that this PR does not solve the first part of the [#27935](https://github.com/PaddlePaddle/Paddle/issues/27935) and currently I'm analyzing solutions to the issue and I'll propose them in the issue's comments.
I really appreciate your suggestion concerning Executor's EnableMKLDNN, it could be legitimate for static  graph execution, however having in mind Dygraph context and not to put too much lines in logs, I'm looking for another solution.

Changes from this PR are beneficial for our internal CI team analyzing logs and could be for Baidu customers as well, in my opinion. Could you accept review of this this PR as standalone one, please?

**Comment by luotao1** [comment]
> Changes from this PR are beneficial for our internal CI team analyzing logs and could be for Baidu customers as well, in my opinion.

@wanghuancoder @phlrain How do you think about it?

**Review by hysunflower — COMMENTED** [review]
@arlesniak  
The resnet and  mobilenet in models/dygraph will be dicarded.
It will be provided in [PaddleClas](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph) repo.

Please contact @littletomatodonkey  for related questions.

**Comment by arlesniak** [comment]
@hysunflower @littletomatodonkey  @luotao1  Thank you for the info. We'll analyze and prepare to switch using resnet and mobilenet models to [PadddleClas](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph). Before we switch fully to the PaddleClas we'll be using this repo and for the meantime could you accept changes from this PR or Baidu prefers it to be closed ?

**Pull request closed without merging**